generic_tracers/generic_tracer.F90

```diff
@@ -262,8 +262,8 @@ subroutine generic_tracer_init(isc,iec,jsc,jec,isd,ied,jsd,jed,nk,ntau,axes,grid
 
     !Allocate and initialize all registered generic tracers
     !JGJ 2013/05/31  merged COBALT into siena_201303
-    if(do_generic_abiotic .or. do_generic_age .or. do_generic_argon .or. do_generic_CFC .or. do_generic_SF6 .or. do_generic_TOPAZ &
-       .or. do_generic_ERGOM .or. do_generic_BLING .or. do_generic_miniBLING .or. do_generic_COBALT .or. do_generic_blres) then
+    if(do_generic_abiotic .or. do_generic_age .or. do_generic_argon .or. do_generic_CFC .or. do_generic_SF6  &
+       .or. do_generic_BLING .or. do_generic_COBALT .or. do_generic_blres) then
```

Comment:
Here the parts for TOPAZ, ERGOM, and miniBLING were deleted rather than commented out, so undoing this PR would not be as simple as un-commenting. To make it possible to revert while also simplifying the code, can we replace this whole statement with `if(do_generic_tracer) then`? The initialization currently gets called for every tracer that is used, and it seems like any other generic tracer that would be added would also need to initialize.